render/detailed_node.go

```diff
@@ -300,6 +303,14 @@ func processOriginTable(nmd report.Node, addHostTag bool, addContainerTag bool)
 	}, len(rows) > 0 || commFound || pidFound
 }
 
+func sparklineRow(human, format string, scale float64, metric report.Metric) Row {
```

Comment:
Wonder if, instead of passing scale & format, we should pass a function (float64 -> string) that does the formatting.  Then have a bunch of standard function, one for percents, one for amounts (kb/mb/gb etc) that picks the right scale.  Thoughts?